docs/pages/cli/verify.mdx

```diff
@@ -0,0 +1,61 @@
+# mud verify
+
+This command uploads the source code for the deployed contracts to a public repository, such as a block explorer.
+The repository then compiles the source code and verifies that the compiled version is identical to what is deployed on-chain.
+
+## Using the command
+
+1. Before you run `mud verify` you need to create a [Foundry profile](https://book.getfoundry.sh/reference/config/overview).
+   Change to the `packages/contract` directory and edit `foundry.toml` to add a profile with the `eth_rpc_url` for your network (if needed).
+   For example, add these lines to add Holesky.
+
+   ```toml copy
+   [profile.holesky]
+   eth_rpc_url = "https://ethereum-holesky.publicnode.com"
+   ```
+
+   Note that this blockchain has to be supported by the verifier.
+
+1. If using Blockscout, you need to specify the API key:
+
+   ```sh copy
+   export ETHERSCAN_API_KEY=<value goes here>
+   ```
```

Comment:
i actually think blockscout doesn't require an api key (but it might be that forge expects _something_ to be provided as an an api key)

Reply:
no blockscout API key afaik, only need to provide an etherscan API key if you use `--verifier etherscan` or no `--verifier` (forge defaults to etherscan)

Reply:
1. We (@yonadaaa and me) decided not to document Etherscan verification at this time because neither of us managed to get it working.

2. If I understand https://docs.blockscout.com/for-users/api#api-keys correctly, it means that Blockscout will serve up to 50 calls without a key every second. Yours may or may not be one of them. But with an API key you get 10 calls a second for yourself.